### DISCUSSION TITLE
RFC to Github issues.

### DISCUSSION THREAD

**Discussion by jmprusi** [operator-framework/operator-controller#340] [General]
Hi folks, we should discuss the process around moving from an accepted RFC to GitHub issues, as now there's nothing defined. 

I will propose a simple template and process as starting point for the discussion:

* Once an RFC is accepted, the responsibles should create a github issue following a similar template:

---

**Title:** [RFC Name/Title]

**Summary**

[Brief overview of what the RFC entails]

**Link to RFC:** [Link to the document, repo, or gist containing the RFC] 

**Related GitHub Issues:** 
- Issue1 
- Issue2
- Issue3

**Dependencies:** 

[ Optional section, to list any tasks or issues outside this RFC that need to be completed before/alongside these tasks.]

- IssueXX  

**Updates and Amendments:** 

[ This section should contain all the changes made to the RFC proposal due decisions/feedback/difficulty during implementation time. If some feature has cut short, or not fully implemented should appear in this section] 

The RFC described this situation, but it is already obsolete due to XX and YY...  

---

* This issue should be assigned to the responsibles that appear in the RFC.

* This issue will serve as an aggregator of all the related tasks/issues that are required to fully implement the linked RFC, making it easy to track/discover the on-going work.

* Once all the linked tasks/issues have been completed and what's described on the RFC implemented, this main issue can be closed.

What do you think?

**Comment by awgreene** [comment]
Should we include a link to the Brief for additional context?

**Reply by jmprusi** [reply]
You can jump to it from the RFC, but we can add another link

**Comment by awgreene** [comment]
Should we remove the checkboxes from the Task/Issues section given that the issues will have a strikethrough format once they're completed? Checkboxes seem to introduce an additional manual step.

**Reply by jmprusi** [reply]
Done!

**Comment by awgreene** [comment]
Can we rename the `Tasks/Issues`  section to `Related GitHub Issues`?

**Reply by jmprusi** [reply]
Done!

**Comment by ncdc** [comment]
I suggest we keep it as simple as possible to minimize effort:

Issue title: RFC name

Issue body:
```
Link to RFC: link

Task list
- [ ] Some task
```

**Comment by awgreene** [comment]
There’s been a number of off discussion proposing that we avoid introducing a template and instead simply let the submitter define the best format, unless I hear opposition, I’m going to close this discussion at end of day.

Oh, please like this comment if you support not adding a template for now.

**Comment by awgreene** [comment]
Closing this to avoid decrease barrier of entry to the project as discussed above.
